build.sbt

```diff
@@ -123,7 +123,18 @@ lazy val compilerPlugin = project.in(file("compiler-plugin"))
     libraryDependencies ++= Seq(
       "com.trueaccord.scalapb" %% "protoc-bridge" % "0.2.7",
       "org.scalatest" %% "scalatest" % "3.0.3" % "test"
-      ))
+    ),
```

Comment:
It looks like this change will shade both the runtime jar and the compiler plugin jar. If I understand correctly, this means that once sbt will upgrade to this version, the issue you are trying to solve will recur since the shaded classes will be in conflict. I suggest to shade only the compiler plugin, and have the ScalaPB version number be part of the prefix name to be safe.

Reply:
My bad, I misread the code and thought that the new settings apply to all the sub-projects and not just the compiler plugin. I made a few minor tweaks and submitted. Thanks for solving this problem!